exercises/space-age/canonical-data.json

```diff
@@ -3,22 +3,22 @@
   "version": "1.3.0",
   "cases": [
     {
-      "description": "age on Earth",
+      "description": "age on Mercury",
```

Comment:
I like the new `x earth years in mercury years` naming scheme. Perhaps we should apply the same scheme to the existing tests?

Reply:
reworked everything as 2 and 20 earth years